kafka-ui-react-app/src/components/Topics/Topic/Details/Details.tsx

```diff
@@ -33,6 +33,7 @@ interface Props extends Topic, TopicDetails {
   topicName: TopicName;
   isInternal: boolean;
   isDeleted: boolean;
+  isDeletePolicy: boolean;
```

Comment:
I don't really like to use booleans like this
Just exposing `cleanUpPolicy` seems fine 
If we need something with other policies - we have to rewrite this or add new `isSomethingPolicy`

Again, it's just a stylistic thing